src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -41,8 +41,8 @@ public function __construct(ContainerInterface $container)
      *
      * @param string    $eventName The name of the event to dispatch. The name of the event is
```

Comment:
Don't we then need to change the description here too?

Reply:
I agree, but out of the scope of this PR. I just want the code to have 0 CS errors. If someone wants to improve the current description, they are more than welcome, but I would not see that as a priority.

Reply:
I agree, I was just wondering why the fixer complained about one case while leaving the other untouched.